test/unit/org/apache/cassandra/index/CustomIndexTest.java

```diff
@@ -162,6 +165,8 @@ public void indexReceivesWriteTimeDeletionsCorrectly() throws Throwable
     @Test
     public void nonCustomIndexesRequireExactlyOneTargetColumn() throws Throwable
     {
+        Assume.assumeTrue("Test does not work with different default secondary index",
```

Comment:
This could be replaced with the new method in Util?

Reply:
Done